unity-client/Assets/Scripts/MainScripts/DCL/Controllers/HUD/EmailPromptHUD/EmailPromptHUDController.cs

```diff
@@ -120,6 +129,7 @@ void OnDismiss(bool dontAskAgain)
 
     void SetEmailFlag()
     {
-        OnSetEmailFlag?.Invoke();
+        // NOTE(Santi): Email prompt set tutorialStep to 128 when finished (this is temporal until we have the new Onboarding flow)
+        WebInterface.SaveUserTutorialStep(UserProfile.GetOwnUserProfile().tutorialStep | 128);
```

Comment:
Should this be updated? based on that comment

Reply:
This instruction must continue existing while the "Email Prompt" feature continue existing in the current way, I mean: Currently the email prompt must be showed to the user 60 seconds after the tutorial is been finished and mark a flag in the profile. In the future, according to what Romi want, this feature will be removed from this tutorial flow and will be part of other different one, and so this part of code could be removed.

Initially the plan was to move out this feature once all the Onboarding features (including the new LogIn/SignIn) were released, but it seems Product don't have still clear where they want to move it and meanwhile it will continue to be in this flow.

Reply:
Maybe we can use a constant to avoid the `128` and the comment.

Reply:
I have created a constant for this value. Thanks for the feedback, guys!